core/space/services/services_sharing.go

```diff
@@ -244,10 +251,13 @@ func (s *Space) ShareFilesViaPublicKey(ctx context.Context, paths *[]domain.Full
 			if err != nil {
 				return err
 			}
-			(*paths)[i].DbId = bs.RemoteDbID
+
+			log.Info("incoming dbid nil, updating")
+			log.Info("using new remote db id: " + bs.RemoteDbID)
```

Comment:
Should these logs be Debug? Seems like they will always be printing when sharing shared-with-me files.

Reply:
These were removed in the next commit :+1: